src/site/content/en/blog/compat2021-midyear/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Compat 2021 mid-year update: Flex gap everywhere"
+description:
+    "Mid-year update on Compat 2021 —an effort to eliminate browser compatibility problems in five 
+    key focus areas: CSS Flexbox, CSS Grid, position: sticky, aspect-ratio, and CSS transforms."
+authors:
+  - foolip
+  - kosamari 
+hero: "image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/KQ5oNcLGKdBSuUM8pFPx.jpeg" alt: "A puzzle
+    with a missing piece." 
+date: 2021-07-23 
+updated: 2021-07-23 
+tags:
+  - blog
+  - CSS
+---
+
+It's time for the mid-year update on Compat 2021—an effort to eliminate browser compatibility
+problems in five key focus areas. For more details about the 
+[#compat2021](https://twitter.com/search?q=%23compat2021) work and how we decided on the 
+areas of focus, check out the [March announcement](https://web.dev/compat2021). 
+
+Improvements to Chromium discussed in this post will reach Chrome, Edge and all Chromium-based
+browsers.
+
+## How we measure progress
+
+You can check the [Compat 2021 dashboard](https://wpt.fyi/compat2021?feature=summary) for
+[web-platform-tests](https://github.com/web-platform-tests/wpt#the-web-platform-tests-project) 
+to see the number of passing tests and the trending graphs for different browsers. 
+
+{% Aside 'key-term' %}
+The web-platform-tests project is a cross-browser test suite for the web platform. The tests are 
+run periodically across multiple browsers and results are available on the 
+[wpt.fyi](https://wpt.fyi/) dashboard.
+{% endAside %}
+
+ A simple "passed tests" number doesn't tell the entire story of browser compatibility, however it
+ is one of the signals we use to see the progress of our effort. Fewer differences between browsers
+ in test results means greater interoperability of a web feature across multiple browsers. 
+
+<figure class="w-figure">
+{% Img src="image/Wd2sVnt4VLho4jgp7UtIyWFceE02/CFL9C7UHKSrAgI5rLrvv.jpg", alt="Caption: a snapshot of Compat 2021 Dashboard (experimental browser types)", width="800", height="538" %}
+  <figcaption class="w-figcaption">
+    A snapshot of <a href="https://wpt.fyi/compat2021">Compat 2021 Dashboard</a> (experimental browser types).
+  </figcaption>
+</figure>
+
+
+## CSS flexbox
```

Comment:
I think lowercasing "flexbox" was suggested by @rachelandrew, but it makes things inconsistent with CSS Grid below (and https://web.dev/compat2021/).

This is not a priority, but if there's a principle behind the capitalization here, I'd like to apply it both here and to the old post.

Reply:
It is a cue from Rachel, there's no "Flexbox" in the spec however "Grid Layout" is capitalized so we've decided to follow the spec formatting. Could you send in a PR to fix the formatting in the previous post?

Reply:
I can send that PR today :D

Reply:
@kosamari no need, Philip already put one in and I just added a few edits.